Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor

```diff
@@ -1,7 +1,7 @@
 ﻿@typeparam TItem
 @inherits BaseComponent
 <Row class="@ClassNames" style="@StyleNames" @attributes="@Attributes">
-    <Column ColumnSize="ColumnSize.IsAuto" Flex="Blazorise.Flex.Row.AlignItems.Center">
+    <Column ColumnSize="ColumnSize.IsAuto" Flex="Blazorise.Flex.Row">

```

Comment:
I believe that was a reason why `.AlignItems.Center` was used? @David-Moreira do you remember why? I think have added it.

Reply:
Ah it was sometime ago, but I remember you refactored it, so it was consistent across providers?

We went back and forth whether we should make it all use flex. And you used AlignItems.Center on your refactor, however there's no mention of the why, but we've tested and it was working fine.

**PR:**
https://github.com/Megabit/Blazorise/pull/5324
**Commit that introduced it:**
https://github.com/Megabit/Blazorise/pull/5324/commits/567ea0031c6b1e26fcf3503f7156fb71a699aeb4

Reply:
Ah yes, I remember. Ok I will see if it makes sense to leave it.